velox/common/file/tests/FileTest.cpp

```diff
@@ -215,3 +215,26 @@ TEST(LocalFile, readFileDestructor) {
     ASSERT_ANY_THROW(readData(&readFile, false));
   }
 }
+
+TEST(LocalFile, mkdir) {
+  filesystems::registerLocalFileSystem();
+  auto tempFolder = ::exec::test::TempDirectoryPath::create();
+
+  std::string path = tempFolder->path;
+  auto localFs = filesystems::getFileSystem(path, nullptr);
+
+  // Create 3 levels of directories and ensure they exist.
+  path += "/level1/level2/level3";
+  EXPECT_TRUE(localFs->mkdir(path));
```

Comment:
```
s/EXPECT_TRUE/ASSERT_TRUE/
```
dittos

Reply:
Why is that?
What is the difference between EXPECT_TRUE and ASSERT_TRUE?

Reply:
You can keep using EXPECT_xxx per our offline discussion. Thanks!